<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoatDialog</class>
 <widget class="QWidget" name="BoatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="priceComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="typLabel">
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="materialLabel">
       <property name="text">
        <string>Material</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="materialComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="besitzComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="besitzLabel">
       <property name="text">
        <string>Besitz</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="herstelldatumDateEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="herstelldatumLabel">
       <property name="text">
        <string>Herstelldatum</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>LÃ¶schen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
